lib/gulp-image-converter.js

```diff
@@ -41,7 +42,44 @@ function convert(options, file, cb) {
   });
 }
 
+function optimizeSvg(svgo, file, cb) {
+  var data = file.contents.toString('utf8');
+
+  // If the SVG was exported from Illustrator with the
+  // "Preserve Illustrator Editing Capabilities" checkbox checked,
+  // it will be very large and use a bunch of proprietary Adobe named
+  // character entities that begin with "&ns_". If we pass this on to svgo,
+  // it will be confused by the entities and throw horribly confusing
+  // errors, so instead we'll detect for the condition here and bail early
+  // with an informative help message.
+  //
+  // For more information, see:
+  //
+  // https://github.com/mozilla/teach.webmaker.org/pull/1052#issuecomment-113582058
+
+  if (data.indexOf('&ns_') !== -1) {
```

Comment:
@toolness ah nvm, just saw this: https://github.com/toolness/teach.webmaker.org/commit/04bff26abbbf28439ecba44c908f64aac84f05a5 :+1: